src/components/ui/Input/Input.tsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * This file is based on the Input component from shadcn and customized for our needs.
+ *
+ * See the official docs for more info:
+ * shadcn/ui: https://ui.shadcn.com/docs/components/input
+ */
+
+import { cn } from '@/lib/tailwind/utils';
+
+import { cva, VariantProps } from 'class-variance-authority';
+import { forwardRef, InputHTMLAttributes } from 'react';
+
+const inputVariants = cva(
+  'flex w-full h-full text-base focus:outline-none disabled:cursor-not-allowed',
+  {
+    variants: {
+      variant: {
```

Comment:
I was thinking of setting the height flexible for user by setting it "h-full".
We can control the size by adding padding in the parent component. 
One of the reason I did that is Because we're not sure yet how much height it will be in desktop design or other designs.

But I'm not sure which is the best practice.

Reply:
I guess that if its parent's paddings control the input size, the input cannot be focused by clicking those paddings, just like what's happening right now in the `SerchInput` component.

Reply:
I think giving `h-full` to the input is an acceptable approach, but in that case, its parent element must have a fixed height without adding paddings.

Reply:
I got it!
Either one of input or parent component has to have fixed height, and other UI components(button, labels) already have fixed size. So maybe it's better to have fixed height in UI components. I'll follow your opinion, thank you!!!

Reply:
`Please consider adding px to the muted and square variants.`


I've included px or the square variants. 
Regarding the muted variant, I believe zero padding is crucial to keep it useful, controllable from parent component.